kernel/packages/decentraland-ecs/src/decentraland/Components.ts

```diff
@@ -513,6 +513,16 @@ export class Animator extends Shape {
     clip.onChange(() => {
       this.dirty = true
     })
+
+    // Access private member to hide internal playCallback implementation to the user
+    clip['playCallback'] = (layer: number) => {
```

Comment:
Make the method public and annotate it with `// @internal` to strip its declaration from the generated bundle. It avoids type unsafety

Reply:
Done. I changed this. Now `AnimationState` has his own owner `Animator` to call that function directly. We decide that for future changes to `Animator`.

Thanks for the info about `// @internal` I didn't know that.